linters/gofumpt/gofumpt.test.ts

```diff
@@ -0,0 +1,6 @@
+import { linterCheckTest, linterFmtTest } from "tests";
+
+// gofumpt generates a failure on an empty file
+linterCheckTest({ linterName: "gofumpt", namedTestPrefixes: ["empty"] });
```

Comment:
Thanks for calling this out. We may move in the future to adding `allow_empty_files: false` to both gofumpt and gofmt, but this should be fine for now.